### PR TITLE
Add Data Table reference docs.

### PR DESCRIPTION
**Pull request by htroisi**

### CODE REVIEW

File: src/functionPageUrl.ts [hunk 1]
```diff
@@ -18,4 +18,4 @@
 
 const slugify = require('slugify');
 
-exports.functionPageUrl = (name, type) => (`/reference/pxl/${slugify(type.toLowerCase())}/${slugify(name.toLowerCase())}`);
+exports.functionPageUrl = (name, type, basepath) => (`${basepath}/${slugify(type.toLowerCase())}/${slugify(name.toLowerCase())}`);
```
Comment (philkuz) [inline src/functionPageUrl.ts]: annoying nit: can you change the order of arguments. The output is basepath, name, type but the input is backwards.

File: src/templates/datatableDocs.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    paddingLeft: '32px',
+    maxWidth: '1280px',
+  },
+  table: {},
+  link: {
+    color: '#12D6D6',
+    textDecoration: 'none',
+  },
+}));
+const RenderFunctionRow = ({ docObj }) => (
```
Comment (philkuz) [inline src/templates/datatableDocs.tsx]:
```suggestion
const RenderTableRow = ({ docObj }) => (
```

File: src/templates/datatableDocs.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                <TableCell align='left'>Column</TableCell>
+                <TableCell align='left'>Type</TableCell>
+                <TableCell align='left'>Pattern</TableCell>
+                <TableCell align='left'>Desc</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {datatables[0].cols
+                .map((f) => (
+                  <RenderFunctionRow docObj={f} />
```
Comment (philkuz) [inline src/templates/datatableDocs.tsx]:
```suggestion
                  <RenderTableRow docObj={f} />
```

File: src/templates/datatableDocsIndex.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    paddingLeft: '32px',
+    maxWidth: '1280px',
+  },
+  table: {},
+  link: {
+    color: '#12D6D6',
+    textDecoration: 'none',
+  },
+}));
+const RenderFunctionRow = ({ docObj, classes, parentPath }) => (
```
Comment (philkuz) [inline src/templates/datatableDocsIndex.tsx]:
```suggestion
const RenderTableRow = ({ docObj, classes, parentPath }) => (
```

File: src/templates/datatableDocsIndex.tsx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+            <TableHead>
+              <TableRow>
+                <TableCell align='left'>Table</TableCell>
+                <TableCell align='left'>Description</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {datatables
+                .map((f) => (
+                  <RenderFunctionRow docObj={f} classes={classes} parentPath={pagePath} />
```
Comment (philkuz) [inline src/templates/datatableDocsIndex.tsx]:
```suggestion
                  <RenderTableRow docObj={f} classes={classes} parentPath={pagePath} />
```

**Review by philkuz — CHANGES_REQUESTED** [review]

**Review by philkuz — APPROVED** [review]

**Review by aimichelle — APPROVED** [review]

**Pull request merged**